server/pkg/model/model_test.go

```diff
@@ -214,12 +214,19 @@ func TestModel_Public(t *testing.T) {
 		want  bool
 	}{
 		{
-			name: "test",
+			name: "set public true",
```

Comment:
```suggestion
			name: "public true",
```

Confusing name; nothing is set.